clients/multiversx/mxClientDataGetter.go

```diff
@@ -238,7 +238,7 @@ func (dataGetter *mxClientDataGetter) ExecuteQueryReturningBigInt(ctx context.Co
 	return num, nil
 }
 
-func parseUInt64FromByteSlice(bytes []byte) (uint64, error) {
+func ParseUInt64FromByteSlice(bytes []byte) (uint64, error) {
```

Comment:
Since now this is being reused, I suggest we move this somewhere in the core package. Also, do not forget to add a comment for it & separate unit tests